recipe/meta.yaml

```diff
@@ -28,8 +28,8 @@ requirements:
     - zlib
     - curl
     - pcre
-    - gshhg-gmt
-    - dcw-gmt
+    - gshhg-gmt  # [not win]
+    - dcw-gmt  # [not win]
```

Comment:
The new Windows GMT 6.1.1 conda package is missing `gshhg-gmt` and `dcw-gmt`, and CI tests are breaking on PyGMT :sweat_smile:  See e.g. https://github.com/GenericMappingTools/pygmt/runs/1398549716?check_suite_focus=true#step:10:457